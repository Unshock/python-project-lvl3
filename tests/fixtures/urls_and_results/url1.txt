https://page-loader.hexlet.repl.co/page/