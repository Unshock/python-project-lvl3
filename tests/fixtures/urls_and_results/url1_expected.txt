page-loader-hexlet-repl-co-page.html